index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+     <nav class="contain">
+      <a href="#" class="navbar-logo">John Doe</a>
+    <div class="humburger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+    </div>
+    </nav>
+    <header class="header-box">
+      <section class="header-content">
+        <h1 class="header-title">Hey There. I'm Abreham</h1>
+        <h2 class="header-subtitle">I’m a Software Developer</h2>
+        <p class="header-text-description">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-media">
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="Images/Icons/github.png" alt="Github icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="Images/Icons/linked.png" alt="Linkedin icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="Images/Icons/angel.png" alt="Angel icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="Images/Icons/twitter.png" alt="Twitter icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="Images/Icons/med.png" alt="medium icon for check"/>
+            </a>
+          </li>
+        </ul>
+      </section>
+    </header>
```

Comment:
After making the changes described above, Kindly remove the header component from here since the header is already defined above 😃